sources/tools/Stride.Importer.3D/Utils.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Silk.NET.Assimp;
+using Stride.Animations;
+using Stride.Core.Mathematics;
+using System.Numerics;
+
+namespace Stride.Importer.ThreeD
+{
+    public static class Utils
+    {
+        public const int AI_MAX_NUMBER_OF_TEXTURECOORDS = 8;
+        public const int AI_MAX_NUMBER_OF_COLOR_SETS = 8;
+
+        public static Matrix ToStrideMatrix(this Matrix4x4 matrix)
+        {
+            // Note the order. Matrices from Assimp has to be transposed
+            return new Matrix(
+                matrix.M11, matrix.M21, matrix.M31, matrix.M41,
+                matrix.M12, matrix.M22, matrix.M32, matrix.M42,
+                matrix.M13, matrix.M23, matrix.M33, matrix.M43,
+                matrix.M14, matrix.M24, matrix.M34, matrix.M44);
+        }
+
+        public static Core.Mathematics.Vector3 ToStrideVector3(this System.Numerics.Vector3 v)
+            => new Core.Mathematics.Vector3(v.X, v.Y, v.Z);
```

Comment:
you can use marshalling like `return Unsafe.As<NumericVector, StrideVector>(ref vec);` to not instantiate a vector all the time as the struct is anyway the same

Reply:
Done! 

This makes sense, to my understanding for this to be fail safe the two have to have exact same layout!